Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -764,6 +843,118 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
 
     #endregion
 
+    #region Quaternion Utils
+
+    /// <summary>
+    /// Converts the quaternion into an axis and an angle and returns the vector
+    /// axis * angle.
+    /// </summary>
+    public static Vector3 ToAngleAxisVector(this Quaternion q) {
+      float angle;
+      Vector3 axis;
+      q.ToAngleAxis(out angle, out axis);
+      return axis * angle;
+    }
+
+    /// <summary>
+    /// Returns a Quaternion described by the provided angle axis vector.
+    /// </summary>
+    public static Quaternion QuaternionFromAngleAxisVector(Vector3 angleAxisVector) {
+      if (angleAxisVector == Vector3.zero) return Quaternion.identity;
+      return Quaternion.AngleAxis(angleAxisVector.magnitude, angleAxisVector.normalized);
+    }
+
+    /// <summary>
+    /// A.From(B) produces the quaternion that rotates from B to A.
```

Comment:
Interesting discussion point, in your explanation of using From, the order is reversed compared to the arguments, and the key operator is _to_ and not from.  Maybe we could entertain adopting that instead?  

`B.To(A)` instead of `A.From(B)`?
`B.Then(B.To(A)) == A` could make sense

thoughts?

Reply:
I generally prefer the From ordering because in "from B", I like to think of the "from" as implying the inversion of B.

To() is a totally reasonable way to think about it, though. I can add To() syntax.

Reply:
I'd probably lean towards only having one, just so we don't have the same concept being represented by multiple methods.  I'm a-ok with using From!

Reply:
nooow you tell me, after I've just gone through and added all the To's

Reply:
Aw man, that's why I called it a discussion!